template-processors/helm/bin/processTemplates.sh

```diff
@@ -24,4 +24,5 @@ helm init --client-only
 helm repo update "${CLONED_TEMPLATE_GIT_DIR}"
 
 echo "Generating manifest files"
-helm template -f /tmp/eunomia_values_processed.yaml --output-dir "${MANIFEST_DIR}" --namespace "${NAMESPACE}" "${CLONED_TEMPLATE_GIT_DIR}"
+helm template -f /tmp/eunomia_values_processed.yaml ${TEMPLATE_PROCESSOR_ARGS:-} \
+  --output-dir "${MANIFEST_DIR}" --namespace "${NAMESPACE}" "${CLONED_TEMPLATE_GIT_DIR}"
```

Comment:
1. In the commit message that introduces the changes here, could you please add a reference to the issue where TEMPLATE_PROCESSOR_ARGS were originally introduced? This could make it easier to understand the bigger context of this change 
2. [optional] What would you think about splitting this command such that each flag+parameter is on a separate line?